src/main/java/command/CommandHandler.java

```diff
@@ -0,0 +1,96 @@
+package command;
+
+import task.TaskBase;
+import task.TaskFactory;
+import task.TaskManager;
+
+public class CommandHandler {
+
+    private TaskManager taskManager;
+    private TaskFactory taskFactory;
+
+    /**
+     * Constructor to create taskManager and taskFactory
+     **/
+    public CommandHandler() {
+        taskManager = new TaskManager();
+        taskFactory = new TaskFactory();
+    }
+
+    /**
+     * Handle command
+     * @param cmd
+     */
+    public void handlerCommand(Command cmd) {
+        CommandType cmdType = cmd.getCommandType();
+        System.out.println("____________________________________________________________");
+
+        if (cmdType == CommandType.LIST) {
+            handleCommandList();
+        } else if (cmdType == CommandType.ADD) {
+            handleCommandAdd(cmd);
+        } else if (cmdType == CommandType.DONE) {
+            handleCommandDone(cmd);
+        } else if (cmdType == CommandType.DELETE) {
+            handleCommandDelete();
+        } else if (cmdType == CommandType.FIND) {
+            handleCommandFind();
+        } else if (cmdType == CommandType.BYE) {
+            handleCommandBye();
+        } else if (cmdType == CommandType.INVALID) {
+            handleCommandInvalid();
+        } else {
+            handleCommandInvalid();
+        }
```

Comment:
Perhaps you can consider using Switch Case instead of if else statements?

Reply:
i second this comment. Switch case here would improve readability